Code/Tools/ProjectManager/Source/ProjectExportWorker.cpp

```diff
@@ -213,7 +225,31 @@ namespace O3DE::ProjectManager
             return AZ::Failure(error);
         }
 
-        return AZ::Success();
+        // Fetch the output directory if the process was successful
```

Comment:
feels odd parsing a log. Export settings also defines the output path right? I guess users could start an export and then half way change the output. Still, maybe there’s a way to save the path in code and use it here?

Reply:
The issue is that they are managed in the o3de python scripts, and there is no direct access to them unless you add c++ functionality to do so, but that is not worth the effort.